frontend/public/components/app-contents.tsx

```diff
@@ -100,13 +107,7 @@ const AppContents = withRouter(React.memo(() => (
 
           <LazyRoute path="/dashboards" exact loader={() => import('./dashboards-page/dashboards' /* webpackChunkName: "dashboards" */).then(m => m.DashboardsPage)} />
           <LazyRoute path="/cluster-status" exact loader={() => import('./cluster-overview' /* webpackChunkName: "cluster-overview" */).then(m => m.ClusterOverviewPage)} />
-          <LazyRoute path="/overview/all-namespaces" exact loader={() => import('./cluster-overview' /* webpackChunkName: "cluster-overview" */).then(m => m.ClusterOverviewPage)} />
-          <LazyRoute path="/overview/ns/:ns" exact loader={() => import('./overview' /* webpackChunkName: "overview" */).then(m => m.OverviewPage)} />
-          <Route path="/overview" exact component={NamespaceRedirect} />
-
-          <LazyRoute path="/status/all-namespaces" exact loader={() => import('./cluster-overview' /* webpackChunkName: "cluster-overview" */).then(m => m.ClusterOverviewPage)} />
-          <LazyRoute path="/status/ns/:ns" exact loader={() => import('./cluster-overview' /* webpackChunkName: "cluster-overview" */).then(m => m.ClusterOverviewPage)} />
-          <Route path="/status" exact component={NamespaceRedirect} />
+          <Route path="/overview" component={OverviewRedirect} />
```

Comment:
I think you could set up the redirect more simply using this format:

https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/Redirect.md#from-string

Reply:
For example,

```jsx
 <Redirect from="/status/ns/:ns" to="/k8s/cluster/projects/:ns/workloads">
```

Reply:
This results in a URL with `:ns` in it.  e.g., `/k8s/cluster/projects/:ns/workloads`  :(  Not sure why.

Reply:
It looks like this is new as of a few days ago. We should try bumping react-router to

https://github.com/ReactTraining/react-router/releases/tag/v4.3.0

Reply:
Bumped `react-router` and used `<Redirects />`